docs/prover/cli/options.md

```diff
@@ -215,6 +215,61 @@ When you have a rule with multiple assertions:
 **Example**
 `certoraRun Bank.sol --verify Bank:Bank.spec --multi_assert_check`
 
+(--independent_satisfy)=
+### `--independent_satisfy`
+
+**What does it do?**
+The independent satisfy mode checks each {ref}`satisfy statement <satisfy>` independently from all other satisfy statements that occurs in a rule. 
+Normally, each satisfy statement will be turned into a sub-rule (similarly to the {ref}`--multi_assert_check` mode), 
+but previously encountered satisfy statements will be still considered when creating a satisfying assignment.
+Turning on the `independent_satisfy` mode will ignore all currently unchecked satisfy statements for each sub-rule.
+
+As an illustrative example, consider the following rule `R` that has two satisfy statements:
+
+```cvl
+rule R {
+  bool b;
+  satisfy b, "R1";
+  satisfy !b, "R2";
+}
+```
+
+The statements for "R1" and "R2" will actually create two sub-rules equivalent to:
+```cvl
+rule R1_default {
+  bool b;
+  satisfy b, "R1";
+}
+
+rule R2_default {
+  bool b;
+  assume b, "R1";
```

Comment:
`assume` is not valid CVL; I think you mean `require`.  The example is a little confusing because the way it is written, the `b && !b` is only showing up in a comment, while the point of showing the examples is to explain the semantics in terms of CVL.  Perhaps if it said "this attempts to produce an example that satisfies both `b` and `!b`" it would be more accurate?

Reply:
I tried to improved the comment by equating it to a satisfy b && !b, "R2"